mev_inspect/compound_liquidations.py

```diff
@@ -35,8 +37,29 @@ def fetch_all_comp_markets(w3: Web3) -> Dict[str, str]:
     return c_token_mapping
 
 
+def fetch_all_cream_markets(w3: Web3) -> Dict[str, str]:
+    cr_token_mapping = {}
+    cream_comptroller_abi = get_raw_abi("Comptroller", Protocol.compound_v2)
+    comptroller_instance = w3.eth.contract(
+        address=CREAM_COMPTROLLER_ADDRESS, abi=cream_comptroller_abi
+    )
+    markets = comptroller_instance.functions.getAllMarkets().call()
+    cream_crtoken_abi = get_raw_abi("CToken", Protocol.compound_v2)
+    for cr_token in markets:
+        # make an exception for cETH (as it has no .underlying())
+        if cr_token != CREAM_CR_ETHER:
```

Comment:
How does this affect things downstream? Will we get a `KeyError` for `CREAM_CR_ETHER` when trying to lookup the collateral address?